CMakeLists.txt

```diff
@@ -239,9 +239,9 @@ find_package(PNG REQUIRED)
 add_definitions(${PNG_DEFINITIONS})
 link_libraries(PNG::PNG)
 
-find_package(JPEG REQUIRED) # -DJPEG_LIBRARY=...
```

Comment:
please leave this line and remove it it from `cmake/FixJPEG.cmake` so we can easier migrate to newer cmake  when needed

Reply:
done.